Issue 3205 : Test and edit Kong Gateway (OSS) Quickstart guide (#3270)
Kong/docs.konghq.com: app/gateway-oss/2.5.x/getting-started/enabling-plugins.md

  </ol>
</div>

> In this section, you're going to learn how to configure Kong plugins. One of the core

Reviewer: ```suggestion
In this section, you'll learn how to configure Kong plugins. One of the core
```

## 2. Verify that the plugin is properly configured

Issue the following cURL request to verify that the [key-auth][key-auth]
plugin is properly configured on the Service:

```bash

> $ curl -i -X GET \

Reviewer: ```suggestion
curl -i -X GET \
```

```

Since you did not specify the required `apikey` header or parameter, the

> response shall be `401 Unauthorized`:

Reviewer: ```suggestion
response should be `401 Unauthorized`:
```
